tests/common

```diff
@@ -843,3 +843,15 @@ function skip_test_no_tpm12()
 		exit 77
 	fi
 }
+
```

Comment:
Please also prefix this patch with `tests:`.

Reply:
ok